geonode_mapstore_client/client/js/components/ActionButtons/ActionButtons.jsx

```diff
@@ -29,7 +29,7 @@ function ActionButtons({
 
 
     return (
-        <div className="gn-resource-action-buttons">
+        !resource.featured ? <div className="gn-resource-action-buttons">
```

Comment:
We could prevent this by not passing the cards menu options in the FeaturedList component instead of this check.
[Here](https://github.com/GeoNode/geonode-mapstore-client/blob/master/geonode_mapstore_client/client/js/routes/Home.jsx#L58) we can try to remove cardOptions prop